pkg/ccl/backupccl/backup_planning.go

```diff
@@ -1728,32 +1736,98 @@ func getBackupDetailAndManifest(
 		DescriptorCoverage:  coverage,
 	}
 
-	// Verify this backup on its prior chain cover its spans up to its end time,
-	// as restore would do if it tried to restore this backup.
-	if err := checkCoverage(ctx, spans, append(prevBackups, backupManifest)); err != nil {
-		return jobspb.BackupDetails{}, BackupManifest{}, errors.Wrap(err, "new backup would not cover expected time")
+	return backupManifest, nil
+}
+
+func updateBackupDetails(
```

Comment:
This function is feeling a little sparse. Maybe it'd be better to have a `makeFullAndShortBackupDetails` and include the filtered version here that's currently created at the end of `getBackupDetailsAndManifest`?

Reply:
I have done this.